File: app/src/main/java/com/zeyad/rxredux/screens/user/list/TopSnappedStickyLayoutManager.java
package com.zeyad.rxredux.screens.user.list;

import com.zeyad.gadapter.stickyheaders.StickyLayoutManager;
import com.zeyad.gadapter.stickyheaders.exposed.StickyHeaderHandler;

import android.content.Context;

final class TopSnappedStickyLayoutManager extends StickyLayoutManager {

    TopSnappedStickyLayoutManager(Context context, StickyHeaderHandler headerHandler) {
        super(context, headerHandler);
    }

    @Override
    public void scrollToPosition(int position) {
        super.scrollToPositionWithOffset(position, 0);
    }
}

File: app/src/main/java/com/zeyad/rxredux/screens/list/TopSnappedStickyLayoutManager.kt
package com.zeyad.rxredux.screens.list

import android.content.Context
import com.zeyad.gadapter.stickyheaders.StickyLayoutManager
import com.zeyad.gadapter.stickyheaders.exposed.StickyHeaderHandler

internal class TopSnappedStickyLayoutManager(context: Context, headerHandler: StickyHeaderHandler) :
        StickyLayoutManager(context, headerHandler) {

    override fun scrollToPosition(position: Int) {
        super.scrollToPositionWithOffset(position, 0)
    }
}
